web/src/pages/Settings/Config/ConfigPanels/LMSMode.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import "../Config.scss";
+import ConfigPanel from './ConfigPanel.jsx';
+import { useStatusStore } from "@/App.jsx";
+import Switch from '@mui/material/Switch/Switch';
+import { IsMobileApp } from "@/utils/MobileApp";
+
+export default function LMSMode() {
+
+    function DownloadConfig() {
+        if (IsMobileApp()) {
+            alert("This feature is not available in the mobile app.");
+            return;
+        }
+        fetch("/api").then((response) => {
+            response.json().then((json) => {
+                const element = document.createElement("a");
+                const d = new Date();
+                const file = new Blob([JSON.stringify(json, undefined, 2)], {
+                    type: "application/json",
+                });
+                element.href = URL.createObjectURL(file);
+                element.download = `amplipi-config-${d.toJSON()}.json`;
+                document.body.appendChild(element);
+                element.click();
+            });
+        });
+    };
+
+    function Base(props) {
+        const { children } = props;
+        return(
+            <div>
+                Lyrion Media Server (LMS) Mode
```

Comment:
Calling attention to this

I noticed that it's called Lyrion in the changelog, thought we should change it here while I'm working here anyways